crawl-ref/source/item-prop.cc

```diff
@@ -793,6 +795,7 @@ const set<pair<object_class_type, int> > removed_items =
     { OBJ_FOOD,      FOOD_ROYAL_JELLY },
     { OBJ_FOOD,      FOOD_UNUSED },
     { OBJ_FOOD,      FOOD_FRUIT },
+	{ OBJ_MISSILES,  MI_SLING_BULLET },
 #endif
```

Comment:
Please indent with four spaces instead of using tabs, for consistency. Tabs render differently on Github and will cause the checkwhite script to fail—this is the reason why Travis has failed.